drivers/sensor/vl53l0x/vl53l0x.c

```diff
@@ -246,9 +246,9 @@ static int vl53l0x_sample_fetch(const struct device *dev,
 	VL53L0X_Error ret;
 	int r;
 
-	__ASSERT_NO_MSG(chan == SENSOR_CHAN_ALL
-			|| chan == SENSOR_CHAN_DISTANCE
-			|| chan == SENSOR_CHAN_PROX);
+	__ASSERT_NO_MSG((chan == SENSOR_CHAN_ALL)
+			|| (chan == SENSOR_CHAN_DISTANCE)
+			|| (chan == SENSOR_CHAN_PROX));
```

Comment:
Please don't add new commits to include suggestions/fixes, but integrate changes directly into impacted commits.

Reply:
Thanks.
I have again rebased on top of the latest origin/main, and squashed this commit with the previous one.